### DISCUSSION TITLE
Product images are not showing

### DISCUSSION THREAD

**Discussion by joyboy67** [solidusio/solidus#4450] [Troubleshooting] [answered]
Hello, I recently debug a troubleshooting from `ruby-vips` gem (https://github.com/solidusio/solidus/discussions/4445) but now my problem is that the product images uploaded are not showed ? In `/storage` I have however folders with the different variants of the images uploaded

**Solidus Version**:

3.2.0.alpha

**Screenshots**

![Screenshot 2022-07-12 at 21-18-59 Images Products](https://user-images.githubusercontent.com/69261996/178681902-ff2da169-d119-4e8d-abf5-53ef89d67ae3.png)

**Desktop (please complete the following information):**

 - OS: OpenBSD 7.1 x64
 - Ruby: 3.1.2p20
 - Rails: 7.0.3
 - MariaDB: mariadb-server-10.6.7p0v1
 - Browser: Firefox

**Comment by jarednorman** [comment]
I've turned this into a discussion instead of an issue, since I don't think there's enough info here to confirm that this is an actual bug with Solidus. This is a better place to get help troubleshooting with your Solidus setup. We can always make issues if the discussion determines that you've found a bug in the framework.

**Comment by jarednorman** [comment]
I'm assuming you're using Active Storage. What storage service do you have configured? (I'm assuming `:local`) What do the URLs are you getting for those images? Is this in production or development?

**Reply by joyboy67** [reply]
Yes I use Active Storage, in local. I got the same result with Vips or Mini Magick

`config/environments/developement.rb` :

```
....
config.active_storage.variant_processor = :vips #mini_magick
config.active_storage.service = :local
....
```

It's in development mode. I uploaded two images for one product, in his product page I got three images URL

http://localhost:3000/rails/active_storage/disk/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaDdDVG9JYTJWNVNTSWhhRGx1WkRkdk5ITXdibUp4TTJkbVl6QTFNWFo1Y21zNE5taDBhUVk2QmtWVU9oQmthWE53YjNOcGRHbHZia2tpUldsdWJHbHVaVHNnWm1sc1pXNWhiV1U5SWpsMlpuaGhhMTlCTG1wd1pXY2lPeUJtYVd4bGJtRnRaU285VlZSR0xUZ25KemwyWm5oaGExOUJMbXB3WldjR093WlVPaEZqYjI1MFpXNTBYM1I1Y0dWSklnOXBiV0ZuWlM5cWNHVm5CanNHVkRvUmMyVnlkbWxqWlY5dVlXMWxPZ3BzYjJOaGJBPT0iLCJleHAiOiIyMDIyLTA3LTE0VDEwOjAwOjUwLjgwMVoiLCJwdXIiOiJibG9iX2tleSJ9fQ==--52ec62d10d5aa1d572a6cc317b680501af373873/9vfxak_A.jpeg

http://localhost:3000/rails/active_storage/disk/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaDdDVG9JYTJWNVNTSWhhRGx1WkRkdk5ITXdibUp4TTJkbVl6QTFNWFo1Y21zNE5taDBhUVk2QmtWVU9oQmthWE53YjNOcGRHbHZia2tpUldsdWJHbHVaVHNnWm1sc1pXNWhiV1U5SWpsMlpuaGhhMTlCTG1wd1pXY2lPeUJtYVd4bGJtRnRaU285VlZSR0xUZ25KemwyWm5oaGExOUJMbXB3WldjR093WlVPaEZqYjI1MFpXNTBYM1I1Y0dWSklnOXBiV0ZuWlM5cWNHVm5CanNHVkRvUmMyVnlkbWxqWlY5dVlXMWxPZ3BzYjJOaGJBPT0iLCJleHAiOiIyMDIyLTA3LTE0VDEwOjAwOjUwLjg2NFoiLCJwdXIiOiJibG9iX2tleSJ9fQ==--5a3bfe3e978c2e9dd923ff80c4a9339af850f676/9vfxak_A.jpeg

http://localhost:3000/rails/active_storage/disk/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaDdDVG9JYTJWNVNTSWhNM056Y1RkbGJIQjBZVzU2YUdVNGVtb3dlbkJ4ZW1KcFoyZHVPQVk2QmtWVU9oQmthWE53YjNOcGRHbHZia2tpUldsdWJHbHVaVHNnWm1sc1pXNWhiV1U5SWxwUGMyazFNVmhaTG1wd1pXY2lPeUJtYVd4bGJtRnRaU285VlZSR0xUZ25KMXBQYzJrMU1WaFpMbXB3WldjR093WlVPaEZqYjI1MFpXNTBYM1I1Y0dWSklnOXBiV0ZuWlM5cWNHVm5CanNHVkRvUmMyVnlkbWxqWlY5dVlXMWxPZ3BzYjJOaGJBPT0iLCJleHAiOiIyMDIyLTA3LTE0VDEwOjAwOjUwLjkxMVoiLCJwdXIiOiJibG9iX2tleSJ9fQ==--da0df4e48e29edab53d23966699bbf04435ebe4f/ZOsi51XY.jpeg

**Comment by joyboy67** [comment, marked as answer]
Thank you @jarednorman, talking about URLs make me found the problem, it's was my fault because in the `config/routes.rb` I added a general redirect `match '*path' => redirect('/'), via: :get`
